### PR TITLE
Changelog: note that GPU support is experimental

### PR DESCRIPTION
**Pull request by sk1p**
## Contributor Checklist:

* [ ] I have added or updated my entry in [the creators.json file](https://github.com/LiberTEM/LiberTEM/blob/master/packaging/creators.json)
* [ ] I have added [a changelog entry](https://github.com/LiberTEM/LiberTEM/tree/master/docs/source/changelog) for my contribution
* [ ] I have added/updated documentation for all user-facing changes
* [ ] I have added/updated test cases
* [ ] I have included the [rebuilt production build of the client](https://libertem.github.io/LiberTEM/contributing.html?#building-the-client) (only if
changes were made to the GUI)

## Reviewer Checklist:

* [ ] `/azp run libertem.libertem-data` passed

### CODE REVIEW

File: docs/source/changelog.rst [hunk 1]
```diff
@@ -26,9 +26,13 @@ We are pleased to announce the latest LiberTEM release, with many
...
 GSoc 2020 students `@AnandBaburajan <https://github.com/AnandBaburajan>`_ and
 `@twentyse7en <https://github.com/twentyse7en>`_, who implemented significant
-improvements in the areas of I/O and the user interface. Another highlight of
-this release is the support of NVidia GPUs, both via CuPy and via native
-libraries. A lot of work was done to implement tiled reading, resulting in a
+improvements in the areas of I/O and the user interface.
+
+Another highlight of this release is experimental support of NVidia GPUs, both
+via CuPy and via native libraries. The API is ready to be used, including support
+in the GUI. Performance optimization is still to be done (:issue:`946`).
```
Comment (uellue) [inline docs/source/changelog.rst:33]: GPU support is activated for all mask-based analyses (virtual detector and Radial Fourier) for testing purposes, but will not bring a noticeable improvement of performance yet. GPU-based processing did show significant benefits for computationally heavy applications like the SSB implementation in https://github.com/Ptychography-4-0/ptychography.

**Comment by uellue** [comment]
Should we merge this or are you still working on it @sk1p ? :-)

**Comment by sk1p** [comment]
Should be ready to merge

**Comment by uellue** [comment]
Thx, merging!

**Pull request merged**